data-collection/deploy/module-securityhub.yaml

```diff
@@ -0,0 +1,311 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Retrieves AWS Security Hub details across AWS organization
+Parameters:
+  DatabaseName:
+    Type: String
+    Description: Name of the Athena database to be created to hold lambda information
+    Default: optimization_data
+  DestinationBucket:
+    Type: String
+    Description: Name of the S3 Bucket that exists or needs to be created to hold costanomaly information
+    AllowedPattern: (?=^.{3,63}$)(?!^(\d+\.)+\d+$)(^(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])$)
+  DestinationBucketARN:
+    Type: String
+    Description: ARN of the S3 Bucket that exists or needs to be created to hold costanomaly information
+  CFDataName:
+    Type: String
+    Description: The name of what this cf is doing.
+    Default: securityhub-events
```

Comment:
should we rather name module securityhub ? 
and have a path like

- securityhub/securityhub-events/

Reply:
renamed module to securityhub